docs/tutorials/setting-up-an-nfs-fileShare.md

```diff
@@ -0,0 +1,180 @@
+
+For general users, it makes it easy to upgrade their SD Cards with known good modules, new kernel `uImage` files, and other updates to config files. It also makes it easier to backup your SD card. Finally, it allows you to edit files on your host machine, then run them on the EV3 without having to actually copy files anywhere!
+
+![nfs_on_ev3dev](Images/Website/nfs_on_ev3dev.png)
+
+It's relatively easy, you just need to keep the addresses and filenames straight. All you have to do is modify one file on the EV3 and one on your host computer. Let's start with getting the computer set up.
+
+The instructions here are for all three platforms Linux, Windows, and OSX - just pick the one you need 
+
+## How To Do It - Linux
+
+On your Linux box, you'll need to edit a file called `/etc/exports`. If you don't have this file, then you need to install `nfs-common` or a similar package. Update the file to look like this:
+
+```
+# /etc/exports: the access control list for filesystems which may be exported
+#		to NFS clients.  See exports(5).
+#
+# Example for NFSv2 and NFSv3:
+# /srv/homes       hostname1(rw,sync,no_subtree_check) hostname2(ro,sync,no_subtree_check)
+#
+# Example for NFSv4:
+# /srv/nfs4        gss/krb5i(rw,sync,fsid=0,crossmnt,no_subtree_check)
+# /srv/nfs4/homes  gss/krb5i(rw,sync,no_subtree_check)
+#
+
+# Share the /home/youruserid/nfs/ev3dev directory on this computer with anyone (*)
+# on the local 192.168.254.* network that asks. If your wifi network assigns a
+# constant IP address to the EV3, then replace the * with the specific address...
+#
+/home/youruserid/nfs/ev3dev   192.168.254.*(rw,sync,no_subtree_check,root_squash)
+```
+
+There may be more lines, of course. All you need to do is tell nfs which directory you want to share (`/home/youruserid/nfs/ev3dev`) and who you want to share it with (`192.168.254.*`). 
+
+Note that `192.168.254.*` is my personal wifi subnet. For those of you that are trying to set up nfs over USB Ethernet, the default subnet that `ev3dev` expects is `192.168.2.*`.
+
+The options, enclosed in parenthesis, tell nfs to:
+
+- Allow read/write access
+- Don't process a write request until the previous write is committed to disk
+- Disables subtree checks, more reliable but mild security risk
+- Do not let root on the EV3 to be like root on the host, good idea!
+
+So update the file on your host machine, then run `sudo exportfs -rv` which will update the directories that nfs exports.
+
+Now update the fstab on the EV3 (see below). 
+
+## How To Do It - Windows
+
+We're in luck! You can download [haneWin NFS](http://www.hanewin.net/nfs-e.htm) which is an nfs server that works on Windows 200x/XP/Vista/7 - hopefully on Windows 8. I have it running on Windows 7 and it's great. The haneWin server has built-in help for setting up the Windows side of the share, and if you get stuck there are [other users on the net that have it working][stmlabshanewin]. 
```

Comment:
It would be helpful to add a note at the top of the section (move the existing one) that warns that this is paid software. Personally, I would recommend against people using this software on Windows because it is a) paid and b) not particularly well-maintained. Having said that, it is a valuable option to document and I am sure that some people will want to use it, so It should still be in here. 

Note that you can add an "alert" on the site using the following Markdown, which should make it more visually apparent:
```
{% include icon.html type="warning" %}
Your text goes here (and <https://example.com>{: .alert-link} links need that extra alert-link notation).
{: .alert .alert-warning}
```